build.sbt

```diff
@@ -1885,13 +1886,49 @@ lazy val searcher = project
   .dependsOn(testkit % Test)
   .dependsOn(`polyglot-api`)
 
+lazy val ydoc = project
+  .in(file("lib/java/ydoc"))
```

Comment:
Why it is called `ydoc`? The library itself has nothing to do with `ydoc`, it is _websocket and other node.js APIs polyfil necessary to execute ydoc on JVM_.

Maybe `ydoc-polyfils`? Or `graal-js-polyfils4ydoc`?